common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -66,6 +76,21 @@ private void addWynncraftButton(TitleScreen titleScreen) {
         titleScreen.addRenderableWidget(wynncraftButton);
     }
 
+    /** Connects to the server on "startup".
+     *  Requires the WynncraftButtonFeature to be enabled.
+     */
+    private void autoConnect() {
+        ServerData wynncraftServer =
+                new ServerData("Wynncraft", connectToLobby.get() ? LOBBY_SERVER : GAME_SERVER, false);
+        wynncraftServer.setResourcePackStatus(ServerData.ServerPackStatus.ENABLED);
```

Comment:
Since this is straight up copied from above, can you extract this into a method and get the `ServerData` instance from there?